### DISCUSSION TITLE
Combo Ports

### DISCUSSION THREAD

**Discussion by wtgitz** [netbox-community/netbox#6668] [Q&A] [answered]
I'm new to Netbox and I'm not sure how to name the combo ports on one of the switches. Two physical ports- one is RJ-45, while the other is SFP; however they are both 25, with SFP having priority if both are active.

Since Netbox doesn't seem to support combo ports, how should I name the interfaces?

**Comment by candlerb** [comment, marked as answer]
If only one can be in use at a time, you can just have a single port "25", and change the interface type depending on how it's being used.

Otherwise, choose whatever naming convention makes sense to you - e.g. "25T" and "25F"
